docs/undeploy.md

```diff
@@ -13,13 +13,12 @@ The `undeploy` task uses a `undeploy` block within the `liberty` block to define
 
 | Attribute | Type  | Since | Description | Required |
 | --------- | ----- | ----- | ----------- | -------- |
-| application| String | 1.0 | Name of the application to be undeployed.| Yes, only when a single application will be undeployed. |
-| include| String | 1.0| Comma- or space-separated list of patterns of files that must be included. All files are included when omitted.| No |
-| exclude| String | 1.0 | Comma- or space-separated list of patterns of files that must be excluded. No files are excluded when omitted.| No |
+| apps | Object[] | 3.0 | Specifies which tasks/files should be undeployed from the `apps` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
```

Comment:
Given that when no apps or dropins property is specified, nothing will be undeployed, I wonder about changing the "Required / No" column for these two properties to the following:

Yes - if `dropins` is not specified.
Yes - if `apps` is not specified.